src/Classes/ServiceAPI/MyRadio_Track.php

```diff
@@ -676,9 +678,11 @@ public static function search(
             'digitised' => filter_var($digitised, FILTER_VALIDATE_BOOLEAN),
             'clean' => $clean,
             'precise' => filter_var($precise, FILTER_VALIDATE_BOOLEAN),
-            'limit' => (($page - 1) * 50) . ',50'
+            'itonesplaylistid' => $itonesplaylistid
         ];
-
+        if ($limit != null) {
```

Comment:
it's 'better' to get do something like get the first 50 and expand (like infinite scrolling, but I hate that too). But meh.